cmake/developer_package/api_validator/api_validator.cmake

```diff
@@ -212,10 +215,3 @@ function(_ov_add_api_validator_post_build_step)
     list(APPEND VALIDATED_TARGETS ${API_VALIDATOR_TARGETS})
     set(VALIDATED_TARGETS "${VALIDATED_TARGETS}" CACHE INTERNAL "" FORCE)
 endfunction()
-
-#
-# ov_add_api_validator_post_build_step(TARGET <name>)
-#
-function(ov_add_api_validator_post_build_step)
-    _ov_add_api_validator_post_build_step(${ARGN})
-endfunction()
```

Comment:
Just curious. What was this extra layer for?

Reply:
I cannot recall..